app/views/admin/users/index.html.haml

```diff
@@ -1,10 +1,14 @@
 .row
   .col-md-12
     .page-header
-      %h2
+      %h1
         Users
         - if @users
           = "(#{@users.length})"
+
+        - if can? :create, User && ENV['OSEM_ICHAIN_ENABLED'] != 'true'
```

Comment:
We should add the rule in abilities

cannot [:new, :create], User if ENV['OSEM_ICHAIN_ENABLED'] == 'true'